pkgs/by-name/ze/zenmap/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchzip,
+  nmap,
+  python3Packages,
+  wrapGAppsHook3,
+  gobject-introspection,
+}:
+
+python3Packages.buildPythonPackage rec {
```

```suggestion
python3Packages.buildPythonApplication rec {
```
Is recommended if you package an application